_content/canon/exalted-utterances_anandajoti.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Exalted Utterances - Udāna"
+translator: anandajoti
```

Comment:
If you're putting in the Canon folder, the filename must be the SuttaCentral ID (i.e. `ud.md`).  The alternative is to put it in `booklets` and make anandajoti an author, which is the solution I prefer in this case.